src/FileSystem-Tests-Core.package/FileSystemTest.class/instance/testCreateFile.st

```diff
@@ -0,0 +1,8 @@
+tests
+testCreateFile
+	| path |
+	path := FileSystem workingDirectory / 'plonk' / 'griffles'.
+	(FileSystem workingDirectory / 'plonk') ensureCreateDirectory.
+	self shouldnt: [ path createFile] raise:Error.
+	self assert:path exists .
+	(FileSystem workingDirectory / 'plonk') deleteAll.
```

Comment:
Should call

`self markForCleanup: path.`

instead of delete the files in the test.